inclusive-design/idrc
# chore: update package dependencies

## Description

Update package dependencies. Also pinpoint package versions to specific versions.

## Steps to test

1. Make sure the build process passes in CI.
2. Make sure the site works the same as before.
